definition-provider-plugin/package.json

```diff
@@ -0,0 +1,39 @@
+{
+      "name": "definition-provider-plugin-plugin",
+      "publisher": "theia",
+      "keywords": [
+        "theia-plugin"
+      ],
+      "version": "0.0.1",
+      "license": "[object Object]",
+
+      "files": [
+        "src"
+      ],
+      "devDependencies": {
+        "@theia/plugin": "next",
+        "@theia/plugin-packager": "latest",
+        "rimraf": "2.6.2",
+        "typescript-formatter": "7.2.2",
+        "typescript": "2.9.2",
+        "ts-loader": "^4.1.0",
+        "clean-webpack-plugin": "^0.1.19",
+        "webpack": "^4.1.1",
+        "webpack-cli": "^3.1.1"
+      },
+      "scripts": {
+        "prepare": "yarn run clean && yarn run build",
+        "clean": "rimraf lib",
+        "format-code": "tsfmt -r",
+        "watch": "webpack-cli -w --config webpack.config.js",
+        "compile": "webpack-cli --config webpack.config.js",
+
+        "build": "yarn run format-code && yarn run compile && theia:plugin pack"
+      },
+      "engines": {
+        "theiaPlugin": "next"
+      },
+      "theiaPlugin": {
+        "frontend": "dist/definition-provider-plugin-frontend-plugin.js"
```

Comment:
why frontend ?

Reply:
Easier to debug

Reply:
I created this plugin and some others for testing of new Theia plugin API methods.

Reply:
Just left one provider to demonstrate abilities of the API.